cherrypy/_cperror.py

```diff
@@ -214,18 +214,7 @@ def __init__(self, urls, status=None, encoding=None):
         if isinstance(urls, text_or_bytes):
```

Comment:
Luckily easy to see by checking the history of the `cherrypy` folder: https://github.com/sabnzbd/sabnzbd/commits/develop/cherrypy

We check very specifically for 8.1.2, so doubt that any packager will not use the included one.